documentation/CAMARA-API-access-and-user-consent.md

```diff
@@ -61,6 +61,8 @@ The list below introduces several key concepts:
 - `Operator`: Mobile Network Operator (MNO), or CSP/telco operator, exposing network capabilities via standard CAMARA APIs.
 - `Aggregator`: aggregate Operator’s CAMARA standardised APIs for building services offered to application developers. An aggregator can be a hyperscaler (e.g. Vonage, AWS, Azure, Google Cloud) offering its own services or directly exposing CAMARA APIs available at the operators, or it can be a telco operator acting as an aggregator, i.e.: aggregating other telco operators and exposing CAMARA APIs available at these telco operators.
 - `API Exposure Platform`: Operator's platform for exposing network capabilities via standard CAMARA APIs. It is the platform that exposes the CAMARA APIs to application developers and provides the authentication and authorization mechanisms to access them. It is also responsible for consent management. It typically consists of at least an Auth Server and an API Gateway.
+- `3-legged access token`: An OAuth2 3-legged access token is a type of token used in the OAuth 2.0 authorisation framework that allows a third-party application to access a user's resources on a resource server on the user's behalf. The term "3-legged" refers to the three parties involved in this flow: the user (the resource owner), the third-party application (the client), and the authorization server/resource server (operator, i.e. service provider).
```

Comment:
```suggestion
- `3-legged access token`: Access tokens are created by the authorization server to be used at the resource server. If the authorization server authenticates the user and potentially asks for their consent for the API access, then the acccess token is called a 3-legged access token, because of the three involved parties: user (the resource owner), authorization server (operator, i.e. service provider), and the client (third-party application). Typically 3-legged access tokens are created through OIDC Authorization Code flow or CIBA.
```

Reply:
Typically 3-legged access tokens are created through OIDC Authorization Code flow or CIBA. -> I've added CAMARA here, because there other flows apart from those ones to obtain 3-legged access tokens but they do not apply to CAMARA.